bb-core/src/main/java/com/cognifide/qa/bb/modules/LoadablesModule.java

```diff
@@ -50,16 +47,18 @@ protected void configure() {
     requestInjection(pageObjectInterceptor);
     WebElementInterceptor webElementInterceptor = new WebElementInterceptor();
     requestInjection(webElementInterceptor);
-    bindInterceptor(subclassesOf(WebElement.class), not(returns(only(ConditionContext.class))), webElementInterceptor);
+    bindInterceptor(subclassesOf(WebElement.class), not(returns(only(ConditionContext.class))),
+        webElementInterceptor);
     bindInterceptor(annotatedWith(PageObject.class), any(), pageObjectInterceptor);
+    Multibinder.newSetBinder(binder(), LoadableProcessorFilter.class);
 
   }
 
   @Override
   protected void bindInterceptor(Matcher<? super Class<?>> classMatcher,
-    Matcher<? super Method> methodMatcher, MethodInterceptor... interceptors) {
```

Comment:
we have following line in **contributing/eclipse-java-bobcat-style.xml**:
```
<setting id="org.eclipse.jdt.core.formatter.continuation_indentation" value="2"/>
```
do we need to update the XML file or is this change invalid?
(nice to have)